modin/pandas/dataframe.py

```diff
@@ -318,8 +318,6 @@ def duplicated(self, subset=None, keep="first"):  # noqa: PR01, RT01, D200
         df = self[subset] if subset is not None else self
         new_qc = df._query_compiler.duplicated(keep=keep)
         duplicates = self._reduce_dimension(new_qc)
-        # remove Series name which was assigned automatically by .apply in QC
-        duplicates.name = None
```

Comment:
Why is it no longer needed?

Reply:
Our `apply` method was assigning a name in the previous implementation. Now it is no longer needed.